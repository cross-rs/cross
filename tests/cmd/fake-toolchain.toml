bin.name = "cross"
args = ["+cross-fake-toolchain", "build"]
status = "failed"
