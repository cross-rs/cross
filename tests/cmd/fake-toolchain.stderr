[..][cross] warning[..]:[..] unable to get metadata for package
[..][cross] note[..]:[..] Falling back to `cargo` on the host.
[..]error[..]:[..] toolchain 'cross-fake-toolchain' is not installed
